### ISSUE TITLE
Debug/run a single TypeScript test from Sidebar fails

### ISSUE THREAD

**Issue by evil-shrike** [maty21/mocha-sidebar#257]
I'm afraid it's a known issue but I haven't found any bug.

**TL;DR;**: adding `--compilers ts:ts-node/register` to command to run a TS test isn't working anymore. It should be replaced with `-r ts-node/register` or simply removed.

### Description
ERROR: --compilers is DEPRECATED and no longer supported.

#### Steps to Reproduce
Open the Sidebar, choose a TS test and click Debug.
Sidebar executes the following command which fails:

```
/Users/segy/homebrew/bin/node ./node_modules/mocha/bin/_mocha src/test/**/*.ts -r ts-node/register --compilers ts:ts-node/register --ui tdd --timeout 5s --grep ^FeedService load files from http load a JSONL file$
```
the error:
ERROR: --compilers is DEPRECATED and no longer supported.

which is quite self-explanatory.

It looks like that Sidebar take a npm "test" script and modifies it with additional ` --grep` arg instead of file mask.
Here's my npm scripts:
```
  "scripts": {
    "clean": "shx rm -rf dist/*",
    "test": "mocha --ui tdd --require ts-node/register --require src/test/fixtures.ts --timeout 10s src/test/**/*.ts",
    "test-coverage": "nyc --reporter=text mocha --ui tdd --require ts-node/register --require src/test/fixtures.ts --timeout 10s src/test/**/*.ts",
    "test-prod": "mocha --ui tdd --require dist/test/fixtures.js --timeout 10s dist/test/**/*.js"
  },
```

### Versions

- VScode: 1.53.2
- Mocha SideBar: v0.22.2
- Node Version: v14.5

### OS version
- [ ] Linux 
- [x] Mac
- [ ] Windows

**Comment by evil-shrike** [comment]
sorry, my bad, the reason is Mocha settings in `.vscode/settings.json`:
```
  "mocha.options": {
>>"compilers": "ts:ts-node/register",
    "ui": "tdd",
    "timeout": "5s"
  },
```
